lib/tasks/convert_application_yml_to_proper_types.rake

```diff
@@ -0,0 +1,66 @@
+namespace :convert_application_yml_to_proper_types do
+  desc 'Take application.yml input file and convert to proper type'
+  task :convert, [:filename] => :environment do |t, args|
+    VALUE_TYPES = ['float', 'string', 'boolean', 'json', 'timestamp', 'integer']
+    filename = args[:filename]
+
+    if filename
+      puts "Making backup copy of '#{filename}' to '#{filename}.bak'"
+      FileUtils.copy(filename, filename+'.bak')
+      content=''
+      add_spaces = false
+      File.open(filename+'.tmp', 'w') { |newfile|
+        IO.foreach(filename) do |l|
+          add_spaces = true if l.include?('development:')
```

Comment:
Is the reason we didn't `YAML.load` this because we want to preserve comments?

Reply:
Yes, YAML.load stripped out comments. I tried that and then said <smh>. So, had to go this direction.